history/people/louisriel.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
# Louis Riel

![Image of Riel](https://upload.wikimedia.org/wikipedia/commons/thumb/2/21/Louis_Riel.jpg/240px-Louis_Riel.jpg)

Considered the _"Father of Manitoba"_, Louis Riel led the Red River Rebellion from 1869 to 1870 and the Northwest Rebellion of 1885 against the Canadian government. These rebellions are credited for preserving the Métis identity. The second rebellion ended in his arrest for high treason and led to his execution. Despite being considered a controversial figure, he is a hero to many Canadians.